packages/2019-disaster-game/src/constants/sfx.js

```diff
@@ -1,63 +1,444 @@
+import thirstInstructionGirlVoice from "../../assets/audio/task_screen/girl/thirst.mp3";
+// Task question audio
+import defaultQuestionGirlVoice from "../../assets/audio/task_screen/girl/how_can_i_help.mp3";
+import rubbleQuestionGirlVoice from "../../assets/audio/task_screen/girl/rubble.mp3";
 
+// SFX
 const BAD_CHOICE = "BAD_CHOICE";
 const GOOD_CHOICE = "GOOD_CHOICE";
 const START_RECORD = "START_RECORD";
+const BADGE_EARNED_SFX = "BADGE_EARNED_SFX";
+const POINTS_EARNED_SFX = "POINTS_EARNED_SFX";
+// Themes
+const THEME_ATTRACTOR = "THEME_ATTRACTOR";
+const THEME_NEUTRAL = "THEME_NEUTRAL";
+const THEME_KIT = "THEME_KIT";
+const THEME_EARTHQUAKE = "THEME_EARTHQUAKE";
+const THEME_TASKS = "THEME_TASKS";
+const THEME_SUMMARY = "THEME_SUMMARY";
+// Voice
+const LETS_PREPARE_BOY = "LETS_PREPARE_BOY";
+const LETS_PREPARE_GIRL = "LETS_PREPARE_GIRL";
+const EARTHQUAKE_BOY = "EARTHQUAKE_BOY";
+const EARTHQUAKE_GIRL = "EARTHQUAKE_GIRL";
+const TASKS_VOTE_INSTRUCTION = "TASKS_VOTE_INSTRUCTION";
+const TASKS_MOTIVATE = "TASKS_MOTIVATE";
+const SUMMARY_MOTIVATION = "SUMMARY_MOTIVATION";
+const SUMMARY_PLAN = "SUMMARY_PLAN";
+const SUMMARY_COMMUNITY = "SUMMARY_COMMUNITY";
+const SUMMARY_KIT = "SUMMARY_KIT";
+// Tasks
+const rubbleInstructionBoy = "rubbleInstructionBoy";
+const dustInstructionBoy = "dustInstructionBoy";
+const hungerInstructionBoy = "hungerInstructionBoy";
+const holeInstructionBoy = "holeInstructionBoy";
+const coldInstructionBoy = "coldInstructionBoy";
+const lostPetInstructionBoy = "lostPetInstructionBoy";
+const injuryInstructionBoy = "injuryInstructionBoy";
+const fireInstructionBoy = "fireInstructionBoy";
+const defaultQuestionBoy = "defaultQuestionBoy";
+const dustQuestionBoy = "dustQuestionBoy";
+const rubbleQuestionBoy = "rubbleQuestionBoy";
+const coldQuestionBoy = "coldQuestionBoy";
+const dustInstructionGirl = "dustInstructionGirl";
+const hungerInstructionGirl = "hungerInstructionGirl";
+const thirstInstructionGirl = "thirstInstructionGirl";
+const defaultQuestionGirl = "defaultQuestionGirl";
+const rubbleQuestionGirl = "rubbleQuestionGirl";
 
-const badChoiceSFX = new Howl({
-  src: [badChoice],
+const defaultHowl = {
   autoplay: false,
   loop: false,
   volume: 1
+};
+
+// SFX
+const badChoiceSFX = new Howl({
+  ...defaultHowl,
+  src: [badChoice]
 });
 
 const goodChoiceSFX = new Howl({
+  ...defaultHowl,
   src: [goodChoice],
-  autoplay: false,
-  loop: false,
   volume: 0.35
 });
 
 const startRecordSFX = new Howl({
-  src: [startRecord],
-  autoplay: false,
-  loop: false,
-  volume: 1
+  ...defaultHowl,
+  src: [startRecord]
 });
 
 const badPhoneSFX = new Howl({
-  src: [badPhone],
-  autoplay: false,
-  loop: false
+  ...defaultHowl,
+  src: [badPhone]
 });
 
 const badBasketballSFX = new Howl({
-  src: [badBasketball],
-  autoplay: false,
-  loop: false
+  ...defaultHowl,
+  src: [badBasketball]
+});
+
+const badgeEarnedSFX = new Howl({
+  ...defaultHowl,
+  src: [badgeEarned],
+  volume: 0.25
+});
```

Comment:
If you're bored (lol) and looking for ways to tighten up the repetitiveness of this file, it might be nice to make a "factory" for these common Howl params.

Something like...

```js
const makeSound = (src, options = {}) => new Howl({ ...defaultHowl, ...options, src: [src] });

const badBasketballSFX = makeSound(badBasketball);
const badgeEarnedSFX = makeSound(badgeEarned, { volume: 0.25 });
```